python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/test_asset_decorator.py

```diff
@@ -105,11 +102,11 @@ def my_third_sling_assets(): ...
 
 
 def test_base_with_meta_config_translator():
-    @sling_assets(
-        replication_config=file_relative_path(
-            __file__, "replication_configs/base_with_meta_config/replication.yaml"
-        )
+    replication_config = file_relative_path(
+        __file__, "replication_configs/base_with_meta_config/replication.yaml"
     )
```

Comment:
No need to assert against the static representation. Just read the file into a dictionary and assert against that.

```suggestion
    replication_config_path = file_relative_path(
        __file__, "replication_configs/base_with_meta_config/replication.yaml"
    )
    replication_config = json.loads(Path(replication_config_path).read_bytes())
```